lib/firebase.dart

```diff
@@ -0,0 +1,64 @@
+import 'package:firebase/firebase.dart' as fb;
+import 'package:firebase/firestore.dart' as fs;
+
+class DB {
+  fs.DocumentReference _filtersRef;
+
+  DB() {
```

Comment:
Let's move this into a config using the same pattern as Coda for how it loads configs - so that we can retarget where it loads the data from and get separation of test and deploy configs

Reply:
Moved this, could you take a look again if it makes sense? @lukechurch Thank you!